tests/bluetooth/controller/mock_ctrl/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+#
+# CMakeLists.txt file for creating of mocks library.
+#
+
+add_library(mocks STATIC
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/kernel.c
```

Comment:
no need for full path, relative sources are picked up relative to `CMAKE_CURRENT_SOURCE_DIR`, so this is sufficient here and below:
```
  src/kernel.c
```

Reply:
done